stable/rabbitmq/values.yaml

```diff
@@ -54,6 +54,12 @@ rabbitmqVhost: /
 ##
 rabbitmqManagerPort: 15672
 
+## RabbitMQ Disk free limit
+## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
+## ref: https://www.rabbitmq.com/disk-alarms.html
+##
+rabbitmqDiskFreeLimit: '"8Gi"'
```

Comment:
This works for you?

Should be 8Gb I think.

However, if you provide a value that rabbit can't parse, it defaults to 50Mb - which will work, but may not be what you want.

Reply:
Yes, it did. No parse error and it removed the disk_free_limit error which returns if I set `rabbitmqDiskFreeLimit='{mem_relative, 1.0}'`. You're right, however, that "Gi" should not work if I look at the [source code]( https://github.com/rabbitmq/rabbitmq-common/blob/master/src/rabbit_resource_monitor_misc.erl).

I will change it to "GiB".

Reply:
Sorry, I missed your comment about the 50Mb fallback. That clarifies, why `"'8Gi'"` magically worked.